main/main.cpp

```diff
@@ -2758,11 +2759,14 @@ bool Main::iteration() {
 	Engine::get_singleton()->_process_frames++;
 
 	if (frame > 1000000) {
+		// Wait a few seconds before printing FPS, as FPS reporting just after the engine has started is inaccurate.
+		print_fps_attempts += 1;
```

Comment:
I refactored the code to avoid incrementing a variable every second.

> Since it's uint32_t, isn't it going to wrap around at some point and then it will skip 4 frames again out of nowhere?

It would, but for an `uint32_t` to overflow when incremented every second, it'd take 4.2 billion seconds to happen :slightly_smiling_face: